implementation/domain/web/jbb-frontend-web/src/main/webapp/WEB-INF/templates/acp/system.html

```diff
@@ -10,6 +10,8 @@
         <dd>- Monitoring measures and calculate statistics on real operation of a board depending on
             the usage by users
         </dd>
+        <dt>Session</dt>
```

Comment:
In left menu - it is, but in list in central space of page it's on 3rd position, under monitoring